Xamarin.MacDev/EntitlementExtensions.cs

```diff
@@ -138,5 +153,17 @@ public static void SetPassBookIdentifiers (this PDictionary dict, PArray value)
 			else
 				dict[EntitlementKeys.PassBookIdentifiers] = value;
 		}
+
+		public static IEnumerable<string> GetEntitlementsKeys (this PDictionary dict)
+		{
```

Comment:
I'd just call this GetEntitlementKeys (i.e. note the lack of an 's' after Entitlement).